### PR TITLE
add jupyter docker image and update integration workflow

### PR DESCRIPTION
**Pull request by MJedr**

### CODE REVIEW

File: docker/jupyter/jupyter_requirements.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# jupyterhub==2.2.1
```
Comment (drjova) [inline docker/jupyter/jupyter_requirements.txt:1]: remove completely?

Comment (MJedr) [reply docker/jupyter/jupyter_requirements.txt:1]: yes, thanks,  a leftover ;)

**Review by drjova — APPROVED** [review]
LGTM, just can we check if there is a way to parallelise that and not to slowdown the rest of the process?

**Comment by MJedr** [comment]
Actually, everything happens after images push, so basically it shouldn't affect any other part of the workflow

**Pull request merged**